README.md

```diff
@@ -122,12 +133,19 @@ const getHtmlKeywords = {
 ##### JS EdgeKit Run
 
 ```typescript
-import { edkt } from '@airgrid/edgekit';
+import { edkt, hasGdprConsent } from '@airgrid/edgekit';
 
+// Vendor ids to check for consent
+const vendorIds = [...]
+
+// Check for GDPR compliance for the provided vendor ids
+const hasConsent = await hasGdprConsent(vendorIds)
+
+// This function won't do anything unless `hasConsent` is `true`
 edkt.run({
   pageFeatureGetters: [getHtmlKeywords],
   audienceDefinitions: ...,
-});
+}, hasConsent);
```

Comment:
I do not like this API, why do we need to pull out an extra function from the internals here and ask the user to execute that, and pass in the response to `edkt`?

Seems more error prone.

Why not just pass in some vendor ids to `run`?

We also spoke about omitting the check by setting a flag.

Reply:
It seemed odd because the wrapper runs the services before calling `edkt.run`, so if it's part of `run` then it would need to call `hasGdprConsent` directly and check for consent earlier and then omit the extra check in `run`. I think you're right though, this makes a lot more sense from an API perspective.